diff --git a/melo/text/__init__.py b/melo/text/__init__.py
index 6e482be..e265501 100644
--- a/melo/text/__init__.py
+++ b/melo/text/__init__.py
@@ -23,13 +23,12 @@ def cleaned_text_to_sequence(cleaned_text, tones, language, symbol_to_id=None):
 def get_bert(norm_text, word2ph, language, device):
     from .chinese_bert import get_bert_feature as zh_bert
     from .english_bert import get_bert_feature as en_bert
-    from .japanese_bert import get_bert_feature as jp_bert
     from .chinese_mix import get_bert_feature as zh_mix_en_bert
-    from .spanish_bert import get_bert_feature as sp_bert
-    from .french_bert import get_bert_feature as fr_bert
-    from .korean import get_bert_feature as kr_bert
 
-    lang_bert_func_map = {"ZH": zh_bert, "EN": en_bert, "JP": jp_bert, 'ZH_MIX_EN': zh_mix_en_bert, 
-                          'FR': fr_bert, 'SP': sp_bert, 'ES': sp_bert, "KR": kr_bert}
+    lang_bert_func_map = {
+            "ZH": zh_bert,
+            "EN": en_bert,
+            "ZH_MIX_EN": zh_mix_en_bert,
+        }
     bert = lang_bert_func_map[language](norm_text, word2ph, device)
     return bert
